test/e2e/tests/dapp-interactions.spec.js

```diff
@@ -81,45 +63,35 @@ describe('Dapp interactions', function () {
       },
       async ({ driver }) => {
         await driver.navigate();
-        await driver.fill('#password', 'correct horse battery staple');
-        await driver.press('#password', driver.Key.ENTER);
-
-        await openDapp(driver);
-        windowHandles = await driver.getAllWindowHandles();
-        extension = windowHandles[0];
-
-        // Lock Account
-        await driver.switchToWindow(extension);
-        await driver.clickElement(
-          '[data-testid="account-options-menu-button"]',
-        );
-        await driver.clickElement({ text: 'Lock', tag: 'div' });
 
-        // Connect to Dapp1
+        // Connect to 2nd dapp => DAPP_ONE
         await openDapp(driver, null, DAPP_ONE_URL);
         await driver.clickElement({ text: 'Connect', tag: 'button' });
-        await driver.waitUntilXWindowHandles(4);
-        windowHandles = await driver.getAllWindowHandles();
+        await driver.waitUntilXWindowHandles(3);
 
-        popup = await driver.switchToWindowWithTitle(
-          'MetaMask Notification',
-          windowHandles,
-        );
+        await driver.switchToWindowWithTitle(WINDOW_TITLES.Notification);
 
-        await driver.switchToWindow(popup);
```

Comment:
Inside this function we will find current all window handlers, so we don't need to have this again
``` 
  let windowHandles =
      initialWindowHandles || (await this.driver.getAllWindowHandles());
```